tests/protocol/stun/attribute-length-ffff.zeek

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
```

Comment:
The format of the weird log has changed between zeek3 and zeek4. I think it would be fine to only enable this test for zeek4, e.g.,

```suggestion

# @TEST_REQUIRES: zeek-version 40000
```

This uses `zeek-version` [from spicy-plugin](https://github.com/zeek/spicy-plugin/blob/main/tests/Scripts/zeek-version).

Reply:
I've added a `zeek-cut uid name addl` step - this should work on either Zeek version.